lucene/core/src/java/org/apache/lucene/index/DocumentsWriterPerThread.java

```
@@ -123,11 +123,7 @@ void abort() {
       if (infoStream.isEnabled("DWPT")) {
         infoStream.message("DWPT", "now abort");
       }
-      try {
-        consumer.abort();
-      } catch (Throwable t) {
-      }
```

Yeah WTF indeed; let it throw and we see the suppressed exc.